examples/styled-jsx-with-csp/pages/_app.jsx

```diff
@@ -0,0 +1,5 @@
+const CustomApp = ({ Component, pageProps }) => <Component {...pageProps} />
+
+CustomApp.getInitialProps = () => ({})
```

Comment:
Is this necessary?

Reply:
Yes, its for disabling next.js static optimisation and make sure the page is always sever rendered cuz `nonce` should be unique on every request.